### DISCUSSION TITLE
Ignore React-native warning logs

### DISCUSSION THREAD

**Discussion by VandaxTheGreat** [community/community#75497] [Programming Help] [answered]
### Body

Is it possible to dismiss or hide the warnings that is logging in react-native, I'm actually planning on creating a tunnel for my test users and I want to avoid them from visibly seeing warning logs. 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by stolenfallen1** [comment, marked as answer]
Yes it is possible but not recommended, tho for your purpose ignoring warning logs is okay, you can try using react-native built in function called LogBox to do so.

```
import { LogBox } from "react-native";

// You can choose the warnings you want to ignore for example
LogBox.ignoreLogs([
  'Sending `onAnimatedValueUpdate` with no listeners registered.', 
  'Possible unhandled promise rejection (id:0: Network request failed)',
  'VirtualizedList: You have a large list that is slow to update',
]);

// Or you can choose to just ignore all by doing this
LogBox.ignoreAllLogs(true);
```

Hope this helps!
